### PR TITLE
Alpha 273 incorrect make/take price in orders with small price value

### PR DESCRIPTION
**Pull request by Rtserg**

### CODE REVIEW

File: order/core/src/test/kotlin/com/rarible/protocol/order/core/service/PriceUpdateServiceTest.kt [hunk 1]
```diff
@@ -98,6 +98,39 @@ internal class PriceUpdateServiceTest {
...
+            Erc1155AssetType(AddressFactory.create(), EthUInt256.TEN),
+            EthUInt256.of(10_000_000)
+        )
+        val takeAsset = Asset(
+            EthAssetType,
+            EthUInt256.ONE
+        )
+        val orderV = createOrderVersion(makeAsset, takeAsset)
+        val calculatedPrice = priceUpdateService.withUpdatedAllPrices(orderV)
+        println()
```
Comment (yakischik) [inline order/core/src/test/kotlin/com/rarible/protocol/order/core/service/PriceUpdateServiceTest.kt:113]: Remove debug code

File: order/core/src/test/kotlin/com/rarible/protocol/order/core/service/PriceUpdateServiceTest.kt [hunk 2]
```diff
@@ -98,6 +98,39 @@ internal class PriceUpdateServiceTest {
...
+            EthUInt256.of(123)
+        )
+        val takeAsset = Asset(
+            Erc1155AssetType(AddressFactory.create(), EthUInt256.TEN),
+            EthUInt256.of(BigInteger.valueOf(100_000))
+        )
+        val orderV = createOrderVersion(makeAsset, takeAsset)
+        val calculatedPrice = priceUpdateService.withUpdatedAllPrices(orderV)
+
+        assertEquals(BigDecimal("1.23E-21"), calculatedPrice.takePrice)
```
Comment (yakischik) [inline order/core/src/test/kotlin/com/rarible/protocol/order/core/service/PriceUpdateServiceTest.kt:131]: We're using assertThat() clause in tests

File: order/core/src/test/kotlin/com/rarible/protocol/order/core/service/PriceUpdateServiceTest.kt (hunk 1 above)
Comment (Rtserg) [reply order/core/src/test/kotlin/com/rarible/protocol/order/core/service/PriceUpdateServiceTest.kt]: fixed

File: order/core/src/test/kotlin/com/rarible/protocol/order/core/service/PriceUpdateServiceTest.kt (hunk 2 above)
Comment (Rtserg) [reply order/core/src/test/kotlin/com/rarible/protocol/order/core/service/PriceUpdateServiceTest.kt]: fixed

**Pull request merged**